shop-info-service/serverless.ts

```diff
@@ -0,0 +1,41 @@
+import type { AWS } from '@serverless/typescript';
+
+import { getProductsList } from '@functions/getProductsList';
+import { getProductById } from '@functions/getProductById';
+
+const serverlessConfiguration: AWS = {
+  service: 'shop-info-service',
```

Comment:
should be called product / product-service

Reply:
I belive it doesn't matter how to call the service